file-icons/emoji-reference.md

```diff
@@ -70,31 +85,32 @@ Java
 
 C
 
-- 🏎 .c / .cpp
-- 🤠 .h / .hpp
+- 🏎 .c | .cpp
+- 🤠 .h | .hpp
 - #️⃣ .cs
+- 👷‍♂️ .csproj
 
 web
 
 - 📘 .ts
 - 📒 .js
-- ⚛️ .tsx / .jsx (react)
-- 💅 .styles.ts / .styles.js (styled-components)
-- 💅 .css / .scss
+- ⚛️ .tsx | .jsx (react)
+- 💅 .css | .scss
 - 🌏 .html
-- 🛢 .db / .sqlite / .sqlite3 / .mysql
-- 🛢 .sql
+- 🛢 .db | .sql | .sqlite | .sqlite3 | .mysql
 - 🍪 .cookie
 - 🗃 .json
-- 🧑‍🔬 .cy.ts / .cy.js
-- 🦮 .helpers.ts / .helpers.js
+
+- 💅 .styles.ts | .styles.js (styled-components)
+- 🌲 .cy.ts | .cy.js (cypress)
+- 🦮 .helpers.ts | .helpers.js ()
```

Comment:
do you know what library is `*.helpers.ts` used for? we can put the name of the package in the `()` like styled-componenets above

Reply:
Not a library, it's just a file structure to move some typescript code from the main away from the react file